src/layouts/ReviewRequest/components/Comments/CommentsDrawer.tsx

```diff
@@ -0,0 +1,165 @@
+        justifyContent="space-between"
+        pb="0.5rem"
+      >
+        <Text overflow="hidden" textOverflow="ellipsis" whiteSpace="nowrap">
+          {commenterName}
+        </Text>
+        <Text whiteSpace="nowrap">{`${date}, ${time}`}</Text>
+      </HStack>
+      {children}
+    </Box>
+  )
+}
+
+export type CommentsDrawerProps = Pick<
+  UseDisclosureReturn,
+  "onClose" | "isOpen"
+>
+
+export const CommentsDrawer = ({ siteName, requestId }: CommentProps) => {
+  const {
+    isOpen: isCommentsOpen,
+    onOpen: onCommentsOpen,
+    onClose: onCommentsClose,
+  } = useDisclosure()
+
+  const { data: commentsData, isLoading: isCommentsLoading } = useGetComments({
+    siteName,
+    requestId,
+  })
+
+  const { mutateAsync: updateReadComments } = useUpdateReadComments()
+  const {
+    mutateAsync: updateNotifications,
+    error: updateNotificationsError,
+  } = useUpdateComments()
+
+  const handleUpdateNotifications = async ({ comment }: CommentFormProps) => {
+    await updateNotifications({ siteName, requestId, message: comment })
+  }
+
+  return (
+    <>
+      <IconButton
+        backgroundColor="background.action.defaultInverse"
+        onClick={() => {
+          onCommentsOpen()
+          // updateReadComments({ siteName, requestId })
+        }}
+        aria-label="Open comments"
+        icon={<BiCommentDetail />}
+        variant="clear"
+        boxShadow="0 0 0 2px var(--chakra-colors-gray-100)"
+      />
+      <Drawer
+        isOpen={isCommentsOpen}
+        onClose={onCommentsClose}
+        placement="right"
+        size="sm"
+      >
+        <DrawerOverlay />
+        <DrawerContent px="1rem">
+          <DrawerCloseButton
+            variant="clear"
+            colorScheme="secondary"
+            top="1.25rem"
+          />
+          <DrawerHeader fontSize="1.5rem" color="text.title.alt">
+            Comments
+          </DrawerHeader>
+          <DrawerBody
+            fontSize="0.875rem"
+            whiteSpace="pre-line"
+            color="secondary.500"
+          >
+            <Stack spacing="2rem" height="100%">
+              <Text color="text.helper">
+                Comments apply to all items in this Review request
+              </Text>
+              <Skeleton isLoaded={!isCommentsLoading} flex={1}>
+                {commentsData && commentsData.length > 0 ? (
+                  commentsData.map((comment) => (
+                    <CommentItem
+                      commenterName={comment.email}
+                      commentTime={comment.createdAt}
+                      isNew={comment.isNew}
+                    >
+                      {comment.message}
+                    </CommentItem>
+                  ))
+                ) : (
+                  <Flex
+                    flex={1}
+                    h="100%"
+                    justifyContent="center"
+                    alignItems="center"
+                  >
+                    <ChatImage />
```

Comment:
nit: lets rename `ChatImage` to `EmptyChatImage` for clarity!

Reply:
Makes sense, https://github.com/isomerpages/isomercms-frontend/pull/1102/commits/05893c1206bb62cdbe40728e2119e7697ef3828b